ui/job-view/details/PinBoard.jsx

```diff
@@ -403,6 +401,8 @@ class PinBoard extends React.Component {
                         ? 'btn-lg selected-job'
                         : 'btn-xs'
                     }`}
+                    role="button"
```

Comment:
There are many elements in these files that might be best changed to a `Button` component using the react-strap library we have installed. So I suggest trying to change spans and divs to buttons while trying to keep the styling of the element as similar as possible.